test suite doesn't pass under OS X

FocusAndPaneIndexTest and PaneOrderingTest in test_workspacebuilder.py fail to pass under OS X for 2 reasons:
1. `/sys` does not exist in OS X
2. `/var` is symlinked to `/private/var` and `pane.get('pane_current_path')` returns the resolved path, so `assertEqual` fails with `AssertionError: u'/private/var' != '/var'`. The same also stands for `/tmp`.

I tried to fix this, either by manually resolving the paths in the test case with `os.path.realpath`, or by changing the paths in the test case and the yaml configuration to `/usr`. In both cases there were some weird issues with running the tests.

When I run one of the failing tests alone, it works with either fix. If I run the whole suite, the test cases failed randomly (not always!) because `pane.get('pane_current_path')` returned either my `cwd` (the root of the repo) or my home directory.

--- a/tmuxp/testsuite/test_workspacebuilder.py
+++ b/tmuxp/testsuite/test_workspacebuilder.py
@@ -413,7 +413,7 @@ class PaneOrderingTest(TmuxTestCase):
       - cd /usr/bin
       - cd /usr
       - cd /sbin
-      - cd /tmp
+      - cd /home
     """
 
     def test_pane_order(self):
@@ -423,7 +423,7 @@ class PaneOrderingTest(TmuxTestCase):
             '/usr/bin',
             '/usr',
             '/sbin',
-            '/tmp'
+            '/home'
         ]
         s = self.session
         sconfig = kaptan.Kaptan(handler='yaml')

